en/physics/physics-group-mask.md

```diff
@@ -145,6 +142,14 @@ By executing `(value >>> 0).toString(2)` in the running environment of JavaScrip
 
 ![View binary](img/mask-all.jpg)
 
+## Collision Matrix
+
+The collision matrix is a further encapsulation of the packet mask configuration, which provides a more unified management and makes it easier to initialize the packet mask configuration without writing any code, and can be configured directly in the editor's project Settings.
+
+However, it should be noted that when the collision matrix is turned on, the mask value can only be updated through the interface related to the collision matrix, and the interface related to `setMask` above will be invalid.
+
+For details, please refer to the [collision matrix Settings](../editor/project/index.md#CollisionMatrix) and [the interface related to the collision matrix](physics-system.md#Interfaces).
```

Comment:
For details, please refer to the [collision matrix Settings](../editor/project/index.md#CollisionMatrix) and [the interface related to the collision matrix](physics-system.md#Interfaces).
-->
For details, please refer to the [collision matrix Settings](../editor/project/index.md#CollisionMatrix) documentation and the [interface related to the collision matrix](physics-system.md#Interfaces) documentation.